webapp/packages/plugin-authentication-administration/src/Administration/Users/Teams/GrantedUsers/GrantedUsersTableHeader/GrantedUsersTableHeader.tsx

```diff
@@ -26,14 +26,14 @@ export const GrantedUsersTableHeader = observer<Props>(function GrantedUsersTabl
   const translate = useTranslate();
 
   return (
-    <header className={s(styles, { header: true }, className)}>
```

Comment:
header is a semantic html element, are we sure we want to change it to just div here?

Reply:
fixed in other similar places also